frontend/src/app/sessions/session/tiling-window-manager/tiling-window-manager.component.html

```diff
@@ -0,0 +1,75 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<div class="flex gap-0.5">
+  <ng-container *ngFor="let session of sessions">
+    <div
+      class="flex flex-col"
+      [ngClass]="{
+        height: (fullscreenService.isFullscreen$ | async) === false,
+        'h-screen': fullscreenService.isFullscreen$ | async
+      }"
+      [style.width]="session.width + 'px'"
+    >
+      <div
+        class="flex items-center justify-between gap-2 rounded-t bg-slate-100 p-1"
+      >
+        <span>
+          {{ session.version?.tool?.name }} {{ session.version?.name }},
+          {{ session.type }}
+          <span *ngIf="session.type === 'readonly'"
+            >(project {{ session.project!.name }})</span
+          >
+          {{ session.id }}
```

Comment:
I wouldn't show the session ID. The ID is mainly used internally and the combination (tool, version, type, project?) is unique.

Reply:
Yes, totally agree. This was just some remaining "debug" code as I tested it with two Jupyter sessions and needed a way to distinguish both.